config/config-network.yaml

```diff
@@ -104,3 +104,7 @@ data:
     # 3. Redirected: The Knative ingress will send a 302 redirect for all
     # http connections, asking the clients to use HTTPS.
     httpProtocol: "Enabled"
+
+    # rolloutDuration contains the minimal duration in seconds over which the
+    # Configuration traffic targets are rolled out to the newest revision.
+    rolloutDuration: "0"
```

Comment:
Does anything else have to be enabled in order to use this? 🤔

Reply:
Not right now.
If we think it's necessary, we can add another feature flag — but having flag to enable flag, whose default value is off, seems too much?